android/src/main/java/flutter/plugins/contactsservice/contactsservice/ContactsServicePlugin.java

```diff
@@ -132,7 +132,7 @@ public void onMethodCall(MethodCall call, Result result) {
         }
         break;
       } case "openExistingContact" :{
-        final Contact contact = Contact.fromMap((HashMap)call.arguments);
+        final Contact contact = Contact.fromMap(((HashMap)call.argument("contact")));
```

Comment:
@engylemure why is this encapsulated in double parentheses?

Reply:
@lukasgit it is not needed sorry for that.